.gitignore

```diff
@@ -33,7 +33,7 @@ simple-client
 # CMake temporary files
 CMakeCache.txt
 CMakeFiles
-*.cmake
+#*.cmake
```

Comment:
i commented this out as a point of discussion as this could cause issues for maintainers, though it could be assumed that a PR should be evaluated and therefore we should not run into an actual issue by ignoring cmake files